packages/eslint-config-nextjs-ts/README.md

```diff
@@ -0,0 +1,39 @@
+# eslint-config-nextjs-ts
+
+This package provides Infinum's ESLint NextJS TypeScript [shareable config](https://eslint.org/docs/developer-guide/shareable-configs.html).
+
+## Usage
+
+1. Install the required packages:
+
+   ```sh
+   npm install --save-dev eslint eslint-plugin-jsx-a11y eslint-plugin-react eslint-plugin-react-hooks eslint-config-next @typescript-eslint/eslint-plugin @typescript-eslint/parser @infinumjs/eslint-config-nextjs-ts
+   ```
+
+2. Extend your ESLint config:
+
+   ```json
+   {
+   	"extends": "@infinumjs/eslint-config-nextjs-ts",
+   	"overrides": [
+   		{
+   			"files": ["*.ts", "*.tsx"],
+   			"parserOptions": {
+   				"project": ["./tsconfig.json"]
+   			}
+   		}
+   	],
+   	"parser": "@typescript-eslint/parser"
+   }
+   ```
+
+## License
+
+The [MIT License](../LICENSE)
+
+## Credits
+
+@infinumjs/eslint-config-nextjs-ts is maintained and sponsored by
+[Infinum](https://www.infinum.com).
+
+<img src="https://infinum.com/infinum.png" width="264">
```

Comment:
@mislavmiocevic Although this was only informational, please ignore it. There have been some suggested changes to this, so we'll wait before we include this to libraries.

Reply:
I think we should inject SVG directly and implement some CSS media queries to support dark mode. Doing it like this is a bit weird because of the extra spacing

Reply:
Agreed. If you have access to this task you can check the proposal https://app.productive.io/1-infinum/tasks/task/2709219?filter=LTE%3D&inbox=true&taskActivityId=40002657

Reply:
@isBatak 🔝